app/controllers/posts_controller.rb

```diff
@@ -30,4 +30,9 @@ def create
       end
     end
   end
```

Comment:
- [OPTIONAL] Please kindly use CanCanCan for Post authorization in your Post Controller.